slicops/ui_api/screen.py

```diff
@@ -303,39 +316,62 @@ def _init():
     )
     _FIELD_DEFAULT = PKDict(
         beam_path=PKDict(
+            type="select",
             choices=_choice_map(slicops.device_db.beam_paths()),
             label="Beam Path",
         ),
-        camera=PKDict(choices=(), label="Camera"),
+        camera=PKDict(
+            type="select",
+            choices=(),
+            label="Camera",
+        ),
+        camera_gain=PKDict(
+            type="text",
+            label="Gain",
+            enabled=True,
+            value=None,
+            visible=True,
+        ),
         color_map=PKDict(
+            type="select",
```

Comment:
I think "type" should be "widget". I don't like "type", because it's overloaded and is a Python builtin. I've been following the PEP8 convention btw to say `type_` when a builtin or keyword is used.